tests/.suitespec.json

```diff
@@ -46,7 +46,9 @@
             "ddtrace/settings/__init__.py",
             "ddtrace/settings/config.py",
             "ddtrace/settings/http.py",
-            "ddtrace/settings/integration.py"
+            "ddtrace/settings/integration.py",
+            "ddtrace/_trace/*",
+            "ddtrace/_trace/processor/*"
```

Comment:
ah doesn't the first line match subfolders? 🤔 The `check_suitespec_coverage` seems to show that path covered